tools/reviewer/aa/commands/InitCommand.java

```diff
@@ -52,6 +57,14 @@ public InitCommand(FileUtils fileUtils, GitRepoFactory gitRepoFactory) {
     this.gitRepoFactory = gitRepoFactory;
   }
 
+  private void cloneRepoIntoHead(String dirName, String repoUrl) {
+    String startupOsPath = fileUtils.joinToAbsolutePath(basePath.get(), "head", dirName);
+    System.out.println("Cloning StartupOS into " + startupOsPath);
```

Comment:
It's now not necessarily StartupOS, so something like s/StartupOS/repoUrl.

Reply:
@oferb fixed in https://github.com/google/startup-os/pull/500/commits/e0915c92017ded4188e955585e1109bab2bd7176